xrviz/utils.py

```diff
@@ -62,3 +62,12 @@ def callback(*events):
     source.param.watch(callback, ['value'], onlychanged=False)
     target = pn.Column(pn.Row(source.name, mark), source)
     return target
+
+
+projections_list = ['RotatedPole', 'Mercator', 'LambertCylindrical',
```

Comment:
Can this list be imported from somewhere? It would be a pain to have to keep it synced with the capabilities of cartopy. I would use alphabetic ordering here (`sorted()`).

Reply:
I have `sorted` the list. I have checked but didn't found any source to import projections list.

Reply:
Unfortunately, it requires importing `crs`, which would be better avoided, if possible. I'm not sure if we've decided to depend upon cartopy/geoviews or not. Does hvplot require them?
```
[p for p in dir(crs) if inspect.isclass(getattr(crs, p)) and issubclass(getattr(crs, p), crs.Projection) and not p.startswith('_')]
```

Reply:
Corrected. Thanks!